spec/convertion_by_format/check_open_html_by_screen_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
```

Comment:
require 'spec_helper` should work fine I think

Reply:
Yes, you right, but ruby mine is not found in the declaration 

![image](https://user-images.githubusercontent.com/60688343/116416903-77bbbe80-a843-11eb-9699-f911393e65a3.png)

### Rubymine linter rule

Reports references in code which can't be resolved to any declaration. Some example references would be a method reference, or a constant reference. Unresolved references are often caused by typos, and can lead to NoMethodErrors, NameErrors and more at runtime.

Reply:
This is known rubymine's linter [bug](https://youtrack.jetbrains.com/issue/RUBY-25223)

Running this file should be ok, especially as other files in this project has same `require 'spec_helper'` line

Reply:
Oh, thank you for the link) Next time I use google.

Reply:
@askonev Actually I've tried to use google to find this bug, but was not able to. But I remember that I reported this issue myself, so this allowed me to find it )